packages/README.md

```diff
@@ -9,9 +9,9 @@ Currently, supported commands include package action commands and statistical co
 
 ###Package action commands
 
-* **load** package_name [package_name] ...
+* **load** package_name sha1_checksum [package_name sha1_checksum] ...
```

Comment:
For linux os, it is quite simply to get a checksum, just using "sha1sum filename". I pointed this out in commit message. Will write this in README.md.